demo/MegEngine/python/demo.py

```diff
@@ -182,7 +182,7 @@ def imageflow_demo(predictor, vis_folder, current_time, args):
     )
     os.makedirs(save_folder, exist_ok=True)
     if args.demo == "video":
-        save_path = os.path.join(save_folder, args.path.split("/")[-1])
+        save_path = os.path.join(save_folder, os.path.split(args.path)[1])
```

Comment:
is `os.path.basename` better ?

Reply:
@FateScript I thought I had issues with unix style paths in windows with os.path.basename but actually it works as expected.

I changed all occurences.